trace_viewer/core/timeline_track_view.html

```diff
@@ -310,7 +317,7 @@
     get listenToKeys_() {
       if (!this.viewport_.isAttachedToDocumentOrInTestMode)
         return false;
-      if (this.activeElement instanceof TracingFindControl)
+      if (document.activeElement instanceof TracingFindControl)
```

Comment:
this.activeElement was never defined anywhere within TimelineTrackView and so this function was originally broken for the cases that I was using it for -- it kept checking if undefined was an instance of TracingFindControl). It should be document.activeElement because it's used everywhere else and it makes sense in this context